src/components/Modals/FiatRamps/fiatRampProviders/juno-pay.ts

```diff
@@ -0,0 +1,55 @@
+import { adapters } from '@shapeshiftoss/caip'
+import axios from 'axios'
+import { getConfig } from 'config'
+import { logger } from 'lib/logger'
+
+import { FiatRampAction, FiatRampAsset } from '../FiatRampsCommon'
+
+const moduleLogger = logger.child({
+  namespace: ['Modals', 'FiatRamps', 'fiatRampProviders', 'JunoPay'],
+})
+
+export async function getJunoPayAssets(): Promise<FiatRampAsset[]> {
+  try {
+    const junoPayData  = await axios.get(getConfig().REACT_APP_JUNOPAY_SUPPORTED_COINS)
+    
+    let junoPayToCurrencyList= junoPayData.data.data.settings.buy.to_currency
+    
+    let junoPayCurrencyList = junoPayToCurrencyList.map((item: any) => {
```

Comment:
Would prefer pluralization 
```suggestion
    let junoPayCurrenciesList = junoPayToCurrencyList.map((item: any) => {
```